content/en/cloud-computing.md

```diff
@@ -7,21 +7,18 @@ tags: ["infrastructure", "fundamental", ""]
 
 ## What it is
 
-Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet. 
-Cloud computing gives users the ability to access and use computing power in a remote physical location. 
-Cloud providers like AWS, GCP, Azure, DigitalOcean, and others all offer third parties 
-the ability to rent access to compute resources in multiple geographic locations.
+Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
+Cloud computing can be classified into public and private cloud depending on whether the cloud services are open to the outside.
```

Comment:
Not sure about "whether cloud services are open to the outside (world)." Isn't it about whether you share actual hardware or not? Public cloud: you have your own virtual resources but share the hardware with other cloud customers. Private cloud: your virtualized environment runs on it's own dedicated machine. (cc @seokho-son @iamNoah1)

Reply:
@CathPag 
In my understanding, sharing underlying hardware of virtual resources is not the factor that differentiates a Public cloud and a Private cloud.

Dedicating or sharing hardware in a cloud computing is another decision point regardless of a Public/Private cloud. For instance, we can request a virtual machine using a dedicated hardware in AWS (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/dedicated-instance.html). 

The following is definition from [NIST](https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-145.pdf)

> Private cloud. The cloud infrastructure is provisioned for exclusive use by a single organization
> comprising multiple consumers (e.g., business units). It may be owned, managed, and
> operated by the organization, a third party, or some combination of them, and it may exist
> on or off premises.

> Public cloud. The cloud infrastructure is provisioned for open use by the general public. It may be
> owned, managed, and operated by a business, academic, or government organization, or
> some combination of them. It exists on the premises of the cloud provider.

Reply:
Hmm...this sounds to me like the (real, not virtualized) hardware is dedicated: "The cloud infrastructure is provisioned for exclusive use..." Where I was wrong is assuming it was owned by a third party exclusively. According to NIST's definition, an organization can have its own private cloud (not really sure how that differs from a regular data center, though, but that's a different topic.)

The AWS example doesn't really apply because it's a VPC, so virtualized hardware, not real hardware.

Reply:
@CathPag 
I guess my confusion is from the dedication of hardware. Both private cloud and public cloud are (generally) based on virtualization technology (making VMs from Physical hardware). There is a hardware dedication concept in virtualization technology. We can dedicate specific hardware to provision VMs or VPCs. 

I agree with "the (real, not virtualized) hardware is dedicated" for exclusive use in a private cloud. 
How about this ?
```suggestion
Cloud computing can be classified into private and public cloud depending on whether the cloud infrastructure is dedicated to an organization for exclusive use or shared for open public services.
```


Regarding
> (not really sure how that differs from a regular data center, though, but that's a different topic.)

Let me share my understanding. A data center is a computing infrastructure consists of huge computing hardware. A data center can be infrastructure of a cloud (i.e., cloud data center), but not necessarily (i.e., on-premises data center).